apollo-core/src/test/java/com/spotify/apollo/core/ServiceImplTest.java

```diff
@@ -578,6 +579,39 @@ public void testSimulateCtrlCWaitingForShutdown() throws Exception {
     }
   }
 
+  @Test
+  public void testExtraModules() throws Exception {
+    Service service = ServiceImpl.builder("test")
+        .build();
+
+    try (Service.Instance instance = service.start(
+        new String[0], ConfigFactory.empty(), ImmutableSet.of(new NoopModule()))) {
+      NoopModule.Noop noop =
```

Comment:
@patriknordlen @mspringare can we reuse the `CountingModuleWithPriority` here and remove the `NoopModule` all togheter?

Reply:
@tommyulfsparre technically we could of course but personally I think it's cleaner to keep them separate as we're using them for different purposes.